gce_workload_cert_refresh/main.go

```diff
@@ -223,9 +227,8 @@ func main() {
 	defer logger.Infof("Done")
 
 	// TODO: prune old dirs
-
 	if err := refreshCreds(); err != nil {
-		logger.Fatalf(err.Error())
+		logger.Errorf("Error refreshCreds: %v", err.Error())
```

Comment:
what is the reason to change from fatal to error? fatal will cause the process to exit with an error code, which will be reflected by the systemd service status, which is why it was chosen.